File: fixtures/conformance/kotlin/switch_case.kt
// when con 3 ramas + else: cada when_entry suma +1 INCLUIDO else (la gramatica no
// distingue la rama else por tipo de nodo; modelo TS/Java, espejo de switch_label).
fun switch_case(x: Int): Int {
    when (x) {
        0 -> return 0
        1 -> return 1
        2 -> return 2
        else -> return 3
    }
    return -1
}

File: fixtures/conformance/c/switch_case.c
/* switch con 3 cases + default: la gramatica NO distingue `case` de `default` por
   tipo (ambos case_statement), asi que default suma +1 (modelo TS/Java). */
int switch_case(int x) {
    switch (x) {
    case 0:
        return 0;
    case 1:
        return 1;
    case 2:
        return 2;
    default:
        return 3;
    }
}
